hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/TokenHandlerTestBase.java

```diff
@@ -79,17 +79,15 @@ public class TokenHandlerTestBase {
     protected final HederaKey freezeHederaKey = asHederaKey(freezeKey).get();
     protected final HederaKey feeScheduleHederaKey = asHederaKey(feeScheduleKey).get();
     protected final HederaKey pauseHederaKey = asHederaKey(A_COMPLEX_KEY).get();
-    protected final EntityNum tokenEntityNum = EntityNum.fromLong(1L);
-    protected final TokenID tokenId =
-            TokenID.newBuilder().tokenNum(tokenEntityNum.longValue()).build();
+    protected final TokenID tokenId = asToken(1L);
     protected final String tokenName = "test token";
     protected final String tokenSymbol = "TT";
     protected final Duration WELL_KNOWN_AUTO_RENEW_PERIOD =
             Duration.newBuilder().seconds(100).build();
     protected final Timestamp WELL_KNOWN_EXPIRY =
             Timestamp.newBuilder().seconds(1_234_567L).build();
     protected final TokenID WELL_KNOWN_TOKEN_ID =
-            TokenID.newBuilder().tokenNum(tokenEntityNum.longValue()).build();
+            TokenID.newBuilder().tokenNum(tokenId.tokenNum()).build();
```

Comment:
```suggestion
            tokenId
```

Just use `tokenId` . No need for conversion

Reply:
Updated, thanks.